### PR TITLE
[Application] Add Resnet Pytorch example @open sesame 01/31 14:00

### PR DESCRIPTION
**Pull request by DonghakPark**
Add Resnet PyTorch Example
- It support only training if user want to use this code, user need to update code with test, validation
- This Example's Dataset : CIFAR100
- This Example's network exactly same with NNtrainer Resnet18 example
- We conduct benchmark test base on this code

Signed-off-by: DongHak Park <donghak.park@samsung.com>

### CODE REVIEW

**Review by baek2sm — APPROVED** [review]
LGTM

File: Applications/Resnet/PyTorch/main.py [hunk 1]
```diff
@@ -0,0 +1,143 @@
...
+import torch.nn as nn
+import torch.nn.functional as F
+
+from torch.utils.data import DataLoader
+from torchvision import datasets
+from torchvision.transforms import ToTensor, transforms
+
+DEVICE = "cpu"
+print(f"Using {DEVICE} device")
+print(f"PyTossrch version: {torch.__version__}")
```
Comment (baek2sm) [inline Applications/Resnet/PyTorch/main.py:21]: There is a typo error in version message of pytorch.

Comment (DonghakPark) [reply Applications/Resnet/PyTorch/main.py:21]: [Fixed] Thank you !!!

**Review by jihochu — APPROVED** [review]
LGTM

**Review by jijoongmoon — APPROVED** [review]
LGTM

**Pull request merged**